### PR TITLE
Make copy changes in homepage

### LINKED ISSUE #4176: Update All Copy on Gitcoin.co Homepage
Update All Copy on Gitcoin Homepage

### Definition of Done

- Updated all copy recommended by Marc Beharry
- Review with Marketing Core team
- Refer to this document for[ final copy](https://docs.google.com/document/d/1EAjRYbJt7J26oQcBGhyXRSMouA6h4m8zWeOFfFj252s/edit)

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
Make copy changes in homepage

##### Refers/Fixes
Fixes #4176 

##### Testing
Screen Shot 2019-05-14 at 7 12 23 PM

### CODE REVIEW

File: app/assets/v2/css/home.css [hunk 1]
```diff
@@ -204,7 +204,14 @@ img.play {
   text-decoration: none;
 }
 
-.products .product a:hover {
+.content-links a {
+  color: #fff;
+  text-decoration: none;
+  border-bottom: 1px solid #fff;
```
Comment (octavioamu) [inline app/assets/v2/css/home.css]: why not just underline instead?

Comment (SaptakS) [reply app/assets/v2/css/home.css]: You are right. I became stupid.

File: app/retail/templates/home/index.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <meta name="description" content="The Gitcoin platform connects freelance developers with open bug bounties or online jobs, paid in crypto (ETH). Leverage a global workforce to quickly complete software development projects and coding jobs."/>
+    <meta name="description" content="The Gitcoin platform connects freelance developers with online jobs solving bug bounties, building features, designing creatives paid in crypto (ETH or any token). Leverage a global workforce to quickly complete software development and coding jobs, and find community funding for your project."/>
```
Comment (danlipert) [inline app/retail/templates/home/index.html]: should this read `and designing creatives`?

Comment (thelostone-mc) [reply app/retail/templates/home/index.html]: fixed it

**Pull request merged**